src/targets/gpu/fuse_ops.cpp

```diff
@@ -717,7 +717,7 @@ struct find_gemm_add
         auto gemm = any_cast<rocblas_gemm<op::dot>>(gemm_ins->get_operator());
 
         // Already fused gemm
-        if(not float_equal(gemm.op.beta, 0))
+        if(not float_equal(gemm.beta, 0))
```

Comment:
@pfultz2  This is conflicting with the design. 

In order to allow this fusion, I have to add beta flag to `rocblas_gemm`, which contains the `dot` op as one of the members.  If i do this fusion, i'll have to allow 3 input arguments to the `dot` operator. If `dot` doesn't have any `beta` field itself then, allowing 3 input args has undefined meaning.